pkg/ir/feature_column.go

```diff
@@ -163,10 +193,15 @@ func (c *SeqCategoryIDColumn) ApplyTo(other *FieldDesc) (FeatureColumn, error) {
 	return &SeqCategoryIDColumn{other, c.BucketSize}, nil
 }
 
+// NumClass returns class number of SeqCategoryIDColumn
+func (c *SeqCategoryIDColumn) NumClass() int64 {
+	return c.BucketSize
+}
+
 // EmbeddingColumn represents `tf.feature_column.embedding_column`
 // ref: https://www.tensorflow.org/api_docs/python/tf/feature_column/embedding_column
 type EmbeddingColumn struct {
-	CategoryColumn FeatureColumn
+	CategoryColumn CategoricalColumn
```

Comment:
Maybe change `CategoryColumn CategoricalColumn` to `CatColumn CategoricalColumn` or change the name `CategoricalColumn` in future updates? `CategoryColumn CategoricalColumn` is hard to distinguish.

Reply:
Changed to `CategoryColumn`.